<!DOCTYPE html>
<html>
	<head>
		<title>python3-zope-exceptions - Details</title>
		<link rel="stylesheet" href="../../assets/css/bootstrap.min.css">
		<link rel="stylesheet" href="../../assets/css/main.css">
		<link href="https://apps.fedoraproject.org/global/fedora-bootstrap-fonts/open-sans.css" type="text/css" rel="stylesheet" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
	</head>
	<body>
		<div class="container">
			<div>
				<div class="row">
					<a href="/" class="col-md-6">
						<div>
							<img src="../../../assets/images/logo.png" style="margin-top: -30px;">
						</div>
					</a>
					<div class="col-md-6">
						
					</div>
				</div>
			</div>

			<hr />

			<h1>
				python3-zope-exceptions
				<small class="text-muted"> in Fedora 34</small>
			</h1>
			<p>
			This page displays detailed information for a specific version of python3-zope-exceptions. You might
			want go back to <a href=".">the main page of python3-zope-exceptions</a>.
			</p>

			<h2>Changelog</h2>
			<table class="table table-striped table-borderless">
				<thead>
					<tr>
						<th scope="col">Date</th>
						<th scope="col">Author</th>
						<th scope="col">Change</th>
					</tr>
				</thead>
				
				<tr scope="row">
					<td>2021-01-27</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 4.0.8-19</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_34_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2020-07-29</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 4.0.8-18</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_33_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2020-05-23</td>
					<td>Miro Hrončok <mhroncok at redhat dot com> - 4.0.8-17</td>
					<td>- Rebuilt for Python 3.9</td>
				</tr>
				
				<tr scope="row">
					<td>2020-01-30</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 4.0.8-16</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_32_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2019-09-20</td>
					<td>Miro Hrončok <mhroncok at redhat dot com> - 4.0.8-15</td>
					<td>- Subpackage python2-zope-exceptions has been removed
  See https://fedoraproject.org/wiki/Changes/Mass_Python_2_Package_Removal</td>
				</tr>
				
				<tr scope="row">
					<td>2019-08-16</td>
					<td>Miro Hrončok <mhroncok at redhat dot com> - 4.0.8-14</td>
					<td>- Rebuilt for Python 3.8</td>
				</tr>
				
				<tr scope="row">
					<td>2019-07-26</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 4.0.8-13</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_31_Mass_Rebuild</td>
				</tr>
				
				<tr scope="row">
					<td>2019-02-02</td>
					<td>Fedora Release Engineering <releng at fedoraproject dot org> - 4.0.8-12</td>
					<td>- Rebuilt for https://fedoraproject.org/wiki/Fedora_30_Mass_Rebuild</td>
				</tr>
				
				<tbody>
				</tbody>
			</table>

			<h2>Files</h2>

			<table class="table table-striped table-borderless">
				<thead>
					<tr>
						<th scope="col">Type</th>
						<th scope="col">Path</th>
					</tr>
				</thead>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/share/doc/python3-zope-exceptions/CHANGES.rst</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/share/doc/python3-zope-exceptions/COPYRIGHT.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/share/doc/python3-zope-exceptions/LICENSE.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/share/doc/python3-zope-exceptions/README.rst</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__/__init__.cpython-39.opt-1.pyc</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__/__init__.cpython-39.pyc</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__/exceptionformatter.cpython-39.opt-1.pyc</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__/exceptionformatter.cpython-39.pyc</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__/interfaces.cpython-39.opt-1.pyc</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__/interfaces.cpython-39.pyc</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__/log.cpython-39.opt-1.pyc</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__/log.cpython-39.pyc</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9.egg-info/PKG-INFO</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9.egg-info/SOURCES.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9.egg-info/dependency_links.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9.egg-info/namespace_packages.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9.egg-info/not-zip-safe</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9.egg-info/requires.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9.egg-info/top_level.txt</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__init__.py</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/__pycache__</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/exceptionformatter.py</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/interfaces.py</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions/log.py</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/share/doc/python3-zope-exceptions</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib/python3.9/site-packages/zope</td>
				</tr>
				
				<tr scope="row">
					<td>f</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9-nspkg.pth</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib/python3.9/site-packages/zope.exceptions-4.0.8-py3.9.egg-info</td>
				</tr>
				
				<tr scope="row">
					<td>d</td>
					<td>/usr/lib/python3.9/site-packages/zope/exceptions</td>
				</tr>
				
				<tbody>
		</div>
	</body>
</html>